packages/calypso-e2e/src/lib/components/comments-component.ts

```diff
@@ -50,7 +50,8 @@ export class CommentsComponent {
 		}
 
 		await commentField.type( comment );
-		await Promise.all( [ this.page.waitForNavigation(), submitButton.click() ] );
```

Comment:
The `this.page.waitForNavigation` was timing out and failing with the following error (in AT sites):

```
    ✕ Post a comment (2159 ms)                                                                                                                                                                                                  
    ○ skipped Like the comment                                                                                                                                                                                                  
    ○ skipped Unlike the comment                                                                                                                                                                                                
                                                                                                                                                                                                                                
  ● Likes (Comment) › Post a comment                                                                                                                                                                                            
                                                                                                                                                                                                                                
    page.waitForNavigation: net::ERR_ABORTED; maybe frame was detached?                                                                                                                                                         
    =========================== logs ===========================                                                                                                                                                                
    waiting for navigation until "load"                                                                                                                                                                                         
    ============================================================                                                                                                                                                                
                                                                                                                                                                                                                                
      52 |              await commentField.type( comment );                                                                                                                                                                     
      53 |              await submitButton.click();                                                                                                                                                                             
    > 54 |              await this.page.waitForNavigation();                                                                                                                                                                    
         |                              ^                                                                                                                                                                                       
      55 |      }                                                                                                                                                                                                               
      56 |                                                                                                                                                                                                                      
      57 |      /**                                                                                                                                                                                                             
                                                                                                                                                                                                                                
      at CommentsComponent.postComment (../../packages/calypso-e2e/src/lib/components/comments-component.ts:54:19)                                                                                                              
      at Object.<anonymous> (specs/published-content/likes__comment.ts:67:3)  
```

Removing it fixes the issue, and the test still passes on simple sites.